server/src/db/migrations/20210303194039-create-tests.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
```

Comment:
I think AP5-100 (?) contains these file changes as well. May want to wait to merge one in until the other is merged in.

Reply:
It does, harder to build the front end with no backend.

Reply:
fair lol